### DISCUSSION TITLE
Need to pull Synergy Disk information but cannot retrieve server profile

### DISCUSSION THREAD

**Discussion by Baskervile-Bytes** [HewlettPackard/POSH-HPOneView#602] [Q&A]
Hi,

I am using the Get-HPOVDriveEnclosureInventory CMDLET to retrieve drive information per drive enclosure across all of our Synergy estate
I can get a list of all of the drives along with the jbod name etc. but can someone please explain to me how to retrieve the server profile per disk along with the RAID configuration for the disk group, or is this not possible?

Many thanks

**Comment by jullienl** [comment]
Have a look at this example

https://github.com/jullienl/HPE-Synergy-OneView-demos/blob/master/Powershell/Compute/Server%20Hardware%20Report.ps1
